zerver/tests/test_subs.py

```diff
@@ -475,7 +475,7 @@ def test_create_web_public_stream(self) -> None:
         ]
 
         # Normal user cannot create web-public streams
```

Comment:
Maybe change this comment to reference the permission, or perhaps better just replace it with a `do_set_realm_property` to set it up before making an assumption about its initial state.

Reply:
Updated the comment. I am not sure whether we need to set the setting value here, and if we want to do we want to set it to `owners only` or something else like `admins only`, though we only check it here for member user only.

Reply:
This seems reasonable.